### PR TITLE
Adds clarifications to the tunneling instructions.

### PR DESCRIPTION
**Pull request by farmas**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,6 +36,8 @@ yarn stop-mock-services
 
 ## Development server with backend services hosted in AWS EC2
 
+- **Important: Tunneling to the data-api does not work currently.**
```
Comment (agagne) [inline README.md]: why does tunneling to data-api not work?

File: README.md [hunk 2]
```diff
@@ -36,6 +36,8 @@ yarn stop-mock-services
...
+- Make sure you have Node v13.
 - Install and configure the [AWS CLI](https://aws.amazon.com/cli/).
 - Install and configure the [AWS helper scripts](https://github.com/uw-it-edm/technical-operations/tree/master/aws-helper-scripts).
```
Comment (agagne) [inline README.md]: Why do we need Node v13?

File: README.md (hunk 1 above)
Comment (farmas) [reply README.md]: I don't know. More configuration might be required, but I don't know enough about that API to know how to get it to work yet.

File: README.md (hunk 2 above)
Comment (farmas) [reply README.md]: The script to substitutes variables in the template needs to load the 'fs' module from node (which is a ES module). Node 13 added that ability by default.

Comment (agagne) [reply README.md]: Should we have a concern about developing on a different version than we support in production? (odd versions are not LTS)

Comment (farmas) [reply README.md]: Created: https://jira.cac.washington.edu/browse/CAB-3887

**Pull request merged**